README.md

```diff
@@ -56,70 +59,92 @@ You can set environment variables in IntelliJ IDEA by going to Run > Edit Config
   - Possible values: Android, IOS, Web.
   - Description: Specifies the platform for testing, whether it is Android, iOS or Web. Must be provided with valid values for the tests to function correctly.
 - Accessibility:
-  - Possible values: true, false, 
-  - Description: indicates whether accessibility features are enabled during testing.The variable can take the value of 'null', in which case the Accessibility report will not be shown.
+  - Possible values: true, false.
+  - Description: indicates whether accessibility features are enabled during testing.The variable can take the value of null or blank, in which case the Accessibility report will not be shown.
 - Browser:
   - Possible values: Chrome, Firefox, Edge.
   - Description: Defines the browser to be used for web testing. The 'Browser' variable can take the value of 'null', in which case the default browser used for testing is 'chrome'.
 - Application:
   - Description: Should be the name of the web or mobile app being tested. Application cannot be null and must be provided with valid values for the tests to function correctly.
-- Resolution: 
+- Resolution:
+  - Possibles values: the possibles values of the environment variable resolution are established in the 'allowedResolution.yaml'.
   - Description: Specifies the screen resolution for web testing. If null or blank, a default resolution of 1024x768 is set.
+- Language:
+  - Possibles values: en-GB, en-US, es-ES, fr-FR.
+  - Description: Specifies the language of the system. If null or blank, the default language is es-ES.
 - Apk:
   - Description: Specifies the path or name of the APK file to be installed and tested on an Android device. The tests work if either the Apk field is set, or both AppPackage and AppActivity fields are provided.
-- AppActivity: 
+- AppActivity:
   - Description: represents the app activity of the app being tested for mobile testing. The tests work if both AppPackage and AppActivity fields are provided.
-- AppPackage: 
+- AppIdentifier:
   - Description: Represents the app package of the app being tested for mobile testing. The tests work if both AppPackage and AppActivity fields are provided.
```

Comment:
app package or bundleId in IOS

Reply:
resaltar que appPackage y AppActivity son de android

Reply:
listo